packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -95,6 +95,22 @@ class _FakeClosedCaptionFile extends ClosedCaptionFile {
 }
 
 void main() {
+  void verifyAppLifeCycle(
+    VideoPlayerController controller, {
+    bool isObserving = true,
```

Comment:
Please make this required instead of having a default value; it's better for tests to be as explicit as possible.